resume-project/src/App.js

```diff
@@ -1,61 +1,84 @@
 import React from 'react';
 
+const CareerComponent = () => {
```

Comment:
general comment:  don't name the component w/ the subfix `Component` since that's redundant